### DISCUSSION TITLE
怎么卸载PCL之前自动安装的Java？

### DISCUSSION THREAD

**Discussion by Atclive** [Meloong-Git/PCL#2677] [提问] [answered]
之前玩过好多版本，也下了一堆java，现在不用了，之前的那么多版本怎么卸掉？

**Comment by kong-que-dnf123** [comment]
开调试模式
随便运行一个mc
在PCL的启动日志中找到如下图的地址
自己进去删（）
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/a095b0e6-832a-4026-9090-d95a7e4ed1cf)

**Reply by Atclive** [reply]
是直接把文件全删了就行吗？

**Reply by kong-que-dnf123** [reply]
那得看你想删啥啊
一个JAVA一个文件夹（看下图）
————————————————以下为个人想法——————————
_文件夹里其他文件是这个JAVA的运行文件啥的
bin里的.exe就是他们的集合
毕竟选择JAVA的时候选择的就是JAVA.exe或者JAVAW.exe啊_
—————————————————————————————————
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/bf11d91f-93c2-4238-a287-b2f150c267a5)

**Comment by DreamUniverse843** [comment, marked as answer]
![image](https://github.com/Hex-Dragon/PCL2/assets/53889721/330557f5-11a5-4d60-a64c-b28f46f896a6)
在设置 > 游戏里让 PCL 列出检测到的所有 Java，然后按路径删除即可。
